README.md

```diff
@@ -137,6 +137,10 @@ For example:
         - `source` link to the weights file. Preferably an url to the weights file.
         - `sha256` SHA256 checksum of the model weight file specified by `source` (see `models` section above for how to generate SHA256 checksum)
         - `[attachments]` weight specific attachments that will be included when generating the model package.
+
+- `[execution_model]` Custom mode for running prediction with this model. For more complex prediction procedures like test time data augmentation that cannot be expressed in the current model configuration. The different execution models should be listed in [supported_formats_and_operations.md#Execution Model](https://github.com/bioimage-io/configuration/blob/master/supported_formats_and_operations.md#execution_model)
```

Comment:
The explanation seems suggest another key name `execution_mode`, otherwise we need to explain it is a model for execution of the current model ;)

Reply:
Yes, I agree. `execution_mode` makes more sense.